packages/logs/src/boot/startLogs.ts

```diff
@@ -55,7 +58,7 @@ export function doStartLogs(
   )
 
   const assemble = buildAssemble(session, configuration, reportError)
-  const batch = startLoggerBatch(configuration)
+  const transport = isEventBridgePresent() ? startLoggerEventBridge() : startLoggerBatch(configuration)
```

Comment:
not super fan of the `transport` naming.
what about something a bit like
```suggestion
let onLogEventCollected 
if (isEventBridgePresent()) {
  const bridge = getEventBridge()
  onLogCollected = (message: Context) => bridge.send('log', message) 
} else {
  const batch = startLoggerBatch(configuration)
  onLogCollected = (message: Context) => batch.add(message) 
}
```

Reply:
To m, It feels a bit convoluted to use and event handler for this case. We loose the intent when reading 
https://github.com/DataDog/browser-sdk/blob/19b1552111f4afe7e5e73fcadc1aa140d8d3b1a0/packages/logs/src/boot/startLogs.ts#L90-L95

Reply:
>  It feels a bit convoluted to use and event handler 

it is still the same callback that transport but we keep the concept of batch.add and bridge.send that is lost in startLogs with your current version.

> We loose the intent

which intent?

Reply:
send the message to whatever process it. Also, worth mentioning that the transport "concept" already exist. `TransportConfiguration`, `transport/` (folder).

Reply:
I feel like `transport` is not an action on its own. Maybe if we had `transport.send()` or `transport.add()` it would be nicer. Or a function named `send` instead of `transport`?